demosplan/DemosPlanCoreBundle/Logic/Procedure/ProcedureService.php

```diff
@@ -2112,7 +2148,9 @@ protected function setAuthorizedUsersToProcedure(Procedure $newProcedure, $bluep
             // T15644: T23583:
             // overwrite authorized users in case of used blueprint is a master-blueprint,
             // to avoid authorizing creators of masterblueprint to this new procedure:
-            if ($blueprint->isMasterTemplate() || $blueprint->isCustomerMasterBlueprint()) {
+            if ($blueprint->isMasterTemplate()
+                || $blueprint->getId() === $blueprint->getCustomer()?->getDefaultProcedureBlueprint()?->getId()
```

Comment:
I would keep this check in `isCustomerMasterBlueprint()`

Reply:
restored in 6f731c5800aeab34e5de2da82e2fb5c1a96e133c